plugins/SedRegex/plugin.py

```diff
@@ -239,8 +235,15 @@ def _replacer_process(self, irc, msg, target, pattern, replacement, count, messa
 
                         subst = axe_spaces(subst)
 
-                        return _("%s meant to say: %s") % \
-                            (messageprefix, subst)
+                        if m.nick == msg.nick:
+                            sed_string = self.registryValue('sedString', msg.channel, irc.network)
+                            return _("%s%s %s") % \
+                                (m.nick, sed_string, subst)
```

Comment:
Please use [template strings](https://docs.python.org/3/library/string.html#template-strings) as val suggested. If you do this with [`ircutils.standardSubstitute`](https://github.com/progval/Limnoria/blob/5357f50bed9a830994faf663416c4b05b21f00b0/src/ircutils.py#L1024) (pass the relevant variables here as `env`) it not only allows you to use more variables that the bot fills in at runtime, but also lets you change the entire output string in a freeform way.

Reply:
I don't fully understand how I should do this (especially why I do not entirely understand the standardSubstitute)

Reply:
Basically you call it as `ircutils.standardSubstitute(irc, msg, templateString, env)`, where templateString is something like `$caller thinks $targetNick meant to say: $subst`, and `env` is a dictionary of template variables to values (e.g. `{'targetNick': m.nick, 'caller': msg.nick, 'replacement': subst}`). You can make the config variable hold the entire template string, and this means you can configure the output however you want. A less verbose `<$targetNick> $subst` is possible too for example

Reply:
Implemented in my latest commit.